### PR TITLE
add shk notebook

### PR DESCRIPTION
**Pull request by slemasne**

### CODE REVIEW

File: examples/features/core-lusid/Sub-holding Keys.ipynb [hunk 1]
```diff
@@ -0,0 +1,783 @@
+{
```
Comment (rizwansaeed) [inline examples/features/core-lusid/Sub-holding Keys.ipynb:1]: <p>I've realised that using the scope on the portfolio code doesn't with lpt as it takes the code from inside the file</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/finbourne/sample-notebooks/pull/155/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='8'/>

Comment (rizwansaeed) [inline examples/features/core-lusid/Sub-holding Keys.ipynb:1]: <p>Maybe add a comment that this is a property on the portfolio for the sub-holding key</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/finbourne/sample-notebooks/pull/155/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='11'/>

Comment (rizwansaeed) [inline examples/features/core-lusid/Sub-holding Keys.ipynb:1]: <p>do we need this? can it be formatted nicer if we do</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/finbourne/sample-notebooks/pull/155/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='16'/>

Comment (slemasne) [reply examples/features/core-lusid/Sub-holding Keys.ipynb:1]: <p>See latest LPT update:</p><p><a href="https://pypi.org/project/lusidtools/" target="_blank">https://pypi.org/project/lusidtools/</a></p><p>In version <strong>0.1.1a72 </strong>we made an update which allows users to pass a DataFrame as well as a file. </p><p>You might need to install latest lusidtools?</p><p> </p><p> </p>

Comment (slemasne) [reply examples/features/core-lusid/Sub-holding Keys.ipynb:1]: <p>Ah. This was a mistake. I meant to delete before I pushed. Removed from latest commit. </p>

Comment (slemasne) [reply examples/features/core-lusid/Sub-holding Keys.ipynb:1]: <p>Sure. done. </p>

**Pull request merged**